interfaces/hydra_endpoints/v0/README.md

```diff
@@ -0,0 +1,57 @@
+# `hydra_endpoints`
+
+## Usage
+
+This relation interface describes the expected behavior of charms claiming to be able to provide or consume a hydra endpoint.
+
+The interface will provide admin and public endpoints, in case any other charm belonging to the IAM bundle requires them both.
+
+## Direction
+
+The interface will consist of a provider and a requirer. The provider is expected to supply its public and admin endpoints,
+while the requirer will just read the information from the application databag.
+
+```mermaid
+flowchart
+    Requirer ----> Provider
+    Provider -- hydra_admin_endpoint, hydra_public_endpoint --> Requirer
+```
+
+## Behavior
+
+Both the requirer and the provider need to adhere to a certain set of criteria to be considered compatible with the interface:
+- The provider must be able to supply admin and public endpoints
+- There are no criteria for the requirer as it's supposed to just consume the relation data.
```

Comment:
those 2 lines aren't necessary as the subbullets below will describe those same behaviors